_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -141,55 +141,58 @@ $shadow: transparentize(palette(grayscale, black-t), .5);
 // ----------------------------
 // #TYPOGRAPHY
 // ----------------------------
+
+$ux-pattern-library-font-path: '../fonts' !default;
```

Comment:
@clintonb, this repo is meant to be clean and pure. We can't really allow exceptions like this, because that opens doors for more exceptions and then we're left with a heavily modified, no-longer-pure library. We've tried really hard to make this library extensible and easy to be themed and are available to answer any questions our documentation doesn't.

As @talbs said, this library isn't really to be merged with anything else, rather be a standalone library with which to build a platform or site. The work you guys are doing is a unique case, and we advise making any unique modifications to the project, rather than to this library.

Does this make sense?